var/spack/repos/builtin/packages/gdb/package.py

```diff
@@ -56,6 +56,7 @@ def install(self, spec, prefix):
         options = ['--prefix=%s' % prefix]
         if '+python' in spec:
             options.extend(['--with-python'])
+        options.append('LDFLAGS={0}'.format(spec['python'].libs.ld_flags))
```

Comment:
This should be indented inside of the if-statement, otherwise Python won't be a dependency.